atlp-rwanda/cabal-phantom-frontend
# #gxudrsTT (frontend)- Signin

#### What does this PR do?
Requests to merge ft-frontend-signin-gxudrsTT into develop branch

#### Description of Task to be completed?

- Appropriate error messages are displayed on invalid inputs (Form validation).

- Appropriate error messages are displayed on unsuccessful attempts 

- Upon successful login, the user should be redirected to their notification page (At this point in time, the notification page might be a black page).

- Skeleton UIs/screen are/is shown when the page loads/is loading.

- The page is responsive

#### How should this be manually tested?

- clone this repo and do `git checkout ft-frontend-signin-gxudrsTT`

- **If you are not registered**, register yourself here [in our backend ](https://phantom-cabal-staging.herokuapp.com/api-docs/#/User/post_api_v1_auth_register)then proceed as below

- Use the password sent to you in the email you registered then

- You can also use the form provided and Sign in at [this deployment](https://cabal-fronte-ft-fronten-kzydxr.herokuapp.com)

- Check the responsiveness of the page by inspecting it and modifying the width of the mobile screen

#### What are the relevant pivotal tracker stories?
[gxudrsTT](https://trello.com/c/gxudrsTT/29-sign-in-to-phantom)

#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2020-11-27 at 08 37 59" src="https://user-images.githubusercontent.com/68019992/100516244-39827c00-318b-11eb-9fd9-dd72247b3fd1.png">
